README.md

```diff
@@ -18,9 +18,12 @@ Data is transformed from these standard formats and loaded into Elasticsearch (t
     - **-t, --transformer** *Default: transformers/[[data file basename]].js* The transformer to use. This converts state-specific data formats to our [Point Data Schema](https://github.com/cfpb/grasshopper/blob/master/docs/point_data_spec.md)
     - **-h, --host** *Default: localhost* The elasticsearch host
     - **-p, --port** *Default: 9200* The elasticsearch port
+    - **-l, --log** *Default: debug* The elasticsearch log level
     - **--index** *Default: address* The elasticsearch index
     - **--type** *Default: point* The elasticsearch type
     - **--profile** *Default: default* The aws credentials profile in ~/.aws/credentials. AWS keys as environment variables will override this setting.')
+    - **--s_srs** *Auto-detects if possible* Source spatial reference system. Needed for untransformed .csv files.
```

Comment:
Yeah, I agree. However, -s_srs is the argument that is passed to ogr2ogr. I would be fine having the loader take --source-srs as the parameter, but you don't get the continuity (have to mentally map --source-srs to -s_srs). That said, I agree it is ugly and will change it if you think the continuity between the arguments isn't too important.